distribution/scripts/jmeter/performance-test-scenarios.sh

```diff
@@ -57,6 +59,7 @@ declare -A test_scenario1=(
 declare -A test_scenario2=(
     [name]="CBRSOAPHeaderProxy"
     [display_name]="CBRSOAPHeaderProxy"
+    [description]="Routing message by reading a SOAP header in message payload"
```

Comment:
Routing the message based on a SOAP header in the message payload

Reply:
Changed